config/flutter/blogs/creating-a-flutter-widget-from-scratch-a9c01c47c630.json

```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "source": "medium",
-  "authorIds": [],
+  "authorIds": "suragch",
```

Comment:
See my comments on your other PR, this format is incorrect.

Reply:
Yes Sorry Sam! I was just trying. Will update this.